CAFAna/Systs/EnergySysts.h

```diff
@@ -592,13 +592,16 @@ namespace ana
 
   extern const NResFD kNResFD;
 
+/*
```

Comment:
This used to exist because some part of the code wanted to be able to know that systs were of a specific type, but another needed them to be specifically a vector<ISyst*>. But maybe that was for some other systematics somewhere else and had been cargo-culted here?

Reply:
I checked were EnergySystVector was used and it wasn't really used except for calling GetEnergySysts(). In that case, I thought it'd just make sense to have that function return a vector of ISyst instead of a structure that is not really needed.